modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesConstants.java

```diff
@@ -35,4 +35,9 @@
      */
     public static final String TENANT_SEPARATOR = "/";
 
+    /**
+     * The prefix which distinguishes a durable subscription.
+     */
+    public static final String DURABLE_SUBSCRIPTION_QUEUE_PREFIX = "carbon:";
+
```

Comment:
The thing is, this is what we get as the identifier for the name of the storage queue for a durable subscription. I'm just dropping it when creting the DLC queue.

Changing "carbon:" to "carbon:durable" for the storage queue requires a large amount of work since we have used this in many places. Besides it'll require having to migrate data when upgrading from previous versions. Chaging this will only improve readability. So, I think leaving it as it is, is best.